src/modules/contribution/ContributionPage.vue

```diff
@@ -95,6 +95,8 @@
       ←{{ t('common.backToHome') }}
     </router-link>
   </div>
+
+  <Footer />
```

Comment:
As you guessed you can just put this into `layout/DashboardLayout.vue` so it shows on all dashboard pages

Reply:
Gotcha, thank you for pointing this out :)